mash_scripts/foreman/2.1.0/foreman-2.1-rhel7-dist.mash

```diff
@@ -0,0 +1,12 @@
+[foreman-2.1-rhel7-dist]
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = foreman-2.1-rhel7-dist
```

Comment:
Why is it `foreman-2.1-rhel7-dist` vs `foreman-2.1-rhel7`?

Reply:
SCL and nonscl tags exist on EL7 and they have to combine to form a single mash target

Reply:
ahh okay, thanks!